### PR TITLE
Use Genialis' Jenkins instead of Travis CI and several other fixes

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,64 +0,0 @@
-sudo: required
```
Comment (dblenkus) [inline .travis.yml]: Remove `.travis.yml` from `.gitignore`.

File: docs/contributing.rst [hunk 2]
```diff
@@ -92,7 +92,13 @@ Run docker::
...
 To run the tests, use::
 
-    ./manage.py test resolwe_bio --parallel=2
+    ./manage.py test resolwe_bio --parallel 2
+
+.. note::
+
+    If you don't specify the number of parallel test processes (i.e. you just
+    use ``--parallel``), Django will run one test process per each core
+    available on the machine.
```
Comment (dblenkus) [inline docs/contributing.rst]: It is not a good idea to run tests with `--parallel` on macOS (and probably Windows), where docker runs in virtual machine and becomes totally unresponsive even with 6 parallel tests (docker's vm has 6 cores and 10gb of ram, so bottle neck is probably network).

File: .travis.yml (hunk 1 above)
Comment (tjanez) [inline .travis.yml]: Fixed.

File: docs/contributing.rst (hunk 2 above)
Comment (tjanez) [inline docs/contributing.rst]: Thanks for the tip. I'll warn users/developers on MacOS and Windows about this.

**Pull request merged**